src/Components/Header/Header.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './Header.css';
+
+const Header = () => {
+  return (
+    <header className='App'>
+      <h1>Tainted Tomatillos</h1>
+      <h2 className='small-title'>Where The Ratings Are Honestly Rotten</h2>
+      <button>Login</button>
+      <input>Search for a Movie</input>
```

Comment:
This line of code is throwing a hackability error.  It isn't the reason we aren't able to connect the files.